pkg/kubelet/cm/dra/manager.go

```diff
@@ -150,6 +151,9 @@ func (m *ManagerImpl) reconcileLoop(ctx context.Context) {
 // for each new resource requirement, process their responses and update the cached
 // containerResources on success.
 func (m *ManagerImpl) PrepareResources(ctx context.Context, pod *v1.Pod) error {
+	defer func(startTime time.Time) {
+		metrics.DRAPrepareResourcesDuration.WithLabelValues().Observe(time.Since(startTime).Seconds())
```

Comment:
Let's include the driver name as label (https://github.com/kubernetes/kubernetes/pull/127146#issuecomment-2338924306).

Reply:
I'll do this for the [DRA GRPC metrics](https://github.com/kubernetes/enhancements/pull/4869#discussion_r1789907966).
I'm still hoping that metrics proposed in this PR are also useful as they can show duration of DRA operations (Kubelet + GRPC). I'm also considering adding error counting metrics.